src/shell/views/SearchPage/SearchPage.tsx

```diff
@@ -197,6 +270,29 @@ export const SearchPage: FC = () => {
     return _results;
   }, [results, params]);
 
+  const renderBody = () => {
```

Comment:
why is this extracted into a function versus being inline with a ternary operator?

Reply:
Just thought it would be a bit cleaner to extract it to a function. I can make it inline instead if you think it's better that way.

Reply:
@shrunyan @agalin920 already updated, moved the code inline